core/arch/arm/kernel/early_ta.c

```diff
@@ -107,6 +109,32 @@ static TEE_Result early_ta_get_size(const struct user_ta_store_handle *h,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result early_ta_get_tag(const struct user_ta_store_handle *h,
+				   uint8_t *tag, unsigned int *tag_len)
+{
+	TEE_Result res = TEE_SUCCESS;
+	void *ctx = NULL;
+
+	*tag_len = TEE_SHA256_HASH_SIZE;
+	if (!tag || *tag_len < TEE_SHA256_HASH_SIZE)
+		return TEE_ERROR_SHORT_BUFFER;
```

Comment:
fix:
```
	if (!tag || *tag_len < TEE_SHA256_HASH_SIZE) {
		*tag_len = TEE_SHA256_HASH_SIZE;
		return TEE_ERROR_SHORT_BUFFER;
	}
	*tag_len = TEE_SHA256_HASH_SIZE;
```

Reply:
Thanks, I managed to make that error a few times.